### ISSUE TITLE
Update site: Add Contribution + Features pages 

### ISSUE THREAD

**Issue by flukeout** [mozilla/thimble.mozilla.org#1967]
I think we can improve our homepage by:

* Showcasing our contributors
* Talking about it's open-source roots & community
* Listing community-built features
* Including information on how to get involved

We can do this with a two tab system maybe, leaving our current homepage as the 'default' tab and adding a "Contribute" or "Get Involved" tab.

We can also use this opportunity to update our style to match with the the recent re-branding.

cc @kristinashu is interested in helping with the design! 👍

**Comment by flukeout** [comment]
I'll get a wireframe goign, and we'll jam on it next week.

**Comment by kristinashu** [comment]
Cool, maybe we can do some brainstorming and sketching first

**Comment by flukeout** [comment]
Yeah, I'll hold off on wireframing until then. Dropping my logo explorations in here though....

<img width="835" alt="image" src="https://cloud.githubusercontent.com/assets/25212/24820230/dc89dd66-1b9c-11e7-8f88-03881bca1f4f.png">

**Comment by flukeout** [comment]
Okay, @kristinashu and I worked through some of the content we'd like to show on the site, here's the related doc:

https://docs.google.com/document/d/1ffSVUJ40-c403x9at8zyWtnH51SYTkYQVOVH-FIulhk/edit

**TLDR**
* Site will have three sections
  * Get started (current homepage)
    * Intro & Gallery
  * Features
    * List of cool features with animated gifs
  * Get Involved
    * Explanation of open source nature of Thimble
    * Links + CTA to get involved
    * Contributor showcase including bios, quotes & features they contributed to

Early mockup of new logo + nav

![image](https://cloud.githubusercontent.com/assets/25212/24934026/311fadcc-1ece-11e7-85fd-bbf24d2fd2da.png)

cc @humphd @gideonthomas

**Comment by jgmac1106** [comment]
Should we wait on new logo until zilla is released or is that thimble name
in zilla?

I can help with any content on the three tabs.

**Comment by flukeout** [comment]
Yeah, the mockup I made is using the official type faces and probably very close to what we'll end up running with. We'll have to find another way to include the current Thimble icon/logo - it doesn't work well within the new brand. We'll see.

Cool, thanks for the offer, I think tomorrow I'll have a better idea of the skeleton layout for the pages so stay tuned!

**Comment by humphd** [comment]
This is happening in #1989, closing here.
